src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Cmdlet/StopAzureSqlElasticPoolActivity.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ElasticPool.Model;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.ElasticPool.Cmdlet
+{
+    [Cmdlet(VerbsLifecycle.Stop, "AzureRmSqlElasticPoolActivity", SupportsShouldProcess = true)]
```

Comment:
@payiAzure please add an `OutputType` to this cmdlet -- if this cmdlet does not return any output, then you should implement the [`PassThru`](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#returning-no-output) parameter

Reply:
@cormacpayne I'm not clear about what is the PassThru parameter and how to implement it? can you please point me to any example?
Since I just follow one existing command to implement this similar one. "StopAzureSqlDatabaseActivity.cs"

Reply:
@cormacpayne .Add OutputType and implement the PassThru parameter.